GliaWidgets/Sources/View/Common/OverlayView.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+class OverlayView: UIView {
```

Comment:
Could we make it `final` class?

Reply:
Added https://github.com/salemove/ios-sdk-widgets/pull/1132/commits/76d7e8d8a4249eba3c3190483504f2d520d6224b